src/OnePlusBot/Base/CustomResult.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Discord.Commands;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnePlusBot.Base
+{
+    public class CustomResult : RuntimeResult
+    {
+
+        public CustomResult(CommandError? error, string reason) : base(error, reason)
+        {
+        }
+
+        public static CustomResult FromError(string reason) 
+            => new CustomResult(CommandError.Unsuccessful, reason);
```

Comment:
Expression body in methods does not improve much readability, Replace with method body if you can